pkg/cri/server/image_verifier/v1/image_verifier.proto

```diff
@@ -0,0 +1,34 @@
+/*
+	Copyright The containerd Authors.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+syntax = "proto3";
+
+package containerd.cri.image_verifier;
+option go_package = "github.com/containerd/containerd/pkg/cri/server/image_verifier/v1;image_verifier";
+
+service image_verifier {
+  rpc VerifyImage(VerifyImageRequest) returns (VerifyImageResponse);
+}
+
+message VerifyImageRequest {
+  string image_name = 1;
+  string image_digest = 2;
```

Comment:
Can be combined into a single string `image_ref`

Reply:
If `image_name` is `index.docker.io/library/nginx:latest` and the resolved `image_digest` is `sha256:2d17cc4981bf1e22a87ef3b3dd20fbb72c3868738e3f307662eb40e2630d4320`, then I think you mean combining these into:
```
image_ref = "index.docker.io/library/nginx:latest@sha256:2d17cc4981bf1e22a87ef3b3dd20fbb72c3868738e3f307662eb40e2630d4320"
```

This works for my particular use case (verifying digests), but there is some loss of information here -- if the digest and image name/tag are combined into one field, there is no way for the verifier plugin to see what image was originally requested. For example, an image verifier plugin implementation that prohibited using the unsafe `latest` tag without a digest would not be able to distinguish an image pull of `nginx` (which resolves to `docker.io/library/nginx:latest`) and `nginx:1.21.6@sha256:2d17cc4981bf1e22a87ef3b3dd20fbb72c3868738e3f307662eb40e2630d4320`, where the former is unsafe but the latter is safe.